README.md

```diff
@@ -251,11 +252,38 @@ sequenceDiagram
         Account->>Hook: isValidSignature(keyHash, data, signature)
         Hook-->>Account: result
     else No hook or no permission
-        Account->>Account: _handleIsValidSignature(keyHash, data, signature)
         Account->>Account: getKey(keyHash)
         Account->>Account: key.verify(_hashTypedData(data.hashWithWrappedType()), signature)
         Account->>Account: Return _1271_MAGIC_VALUE or _1271_INVALID_VALUE
     end
     
     Account-->>-VerifyingContract: result (0x1626ba7e if valid, otherwise 0xffffffff)
-```
\ No newline at end of file
+```
+
+## Migration Notes
+
+### Breaking Changes in v1.x.x
+
+The contract has undergone a structural refactoring to improve modularization and separation of concerns:
+
+1. **Structure Changes**:
+   - `SignedCalls` has been replaced with two separate structures:
+     - `BatchedCall`: Contains the call array and shouldRevert flag
+     - `SignedBatchedCall`: Contains a BatchedCall, nonce, and keyHash
+
+2. **Field Access Changes**:
```

Comment:
do we need this? lol seems overkill to update pr per changes?

Reply:
Yea true